src/librbd/CMakeLists.txt

```diff
@@ -265,6 +223,56 @@ set_target_properties(librbd_plugin_parent_cache PROPERTIES
 install(TARGETS librbd_plugin_parent_cache DESTINATION ${librbd_plugins_dir})
 add_dependencies(librbd_plugins librbd_plugin_parent_cache)
 
+set(rbd_plugin_pwl_srcs
```

Comment:
Nit: this whole section should be surrounded by `WITH_RBD_RWL` / `WITH_RBD_SSD_CACHE`

Reply:
DONE